### PR TITLE
Addresses https://jira.duraspace.org/browse/ISLANDORA-1849

### PR DESCRIPTION
**Pull request by ruebot**
**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-1849

# What does this Pull Request do?

Allows Islandora XML Forms to be compatible with PHP 7

# What's new?
Some curly braces.

# How should this be tested?

Run Islandora under PHP 7, and edit the metadata on an object, click save, and you should get an error on the screen, the log should read something along the lines of:

```
Error: Call to undefined function createXML() in Create->doCreate() (line 193 of /var/www/digital.library.yorku.ca/sites/all/modules/islandora_xml_forms/api/Create.inc).
```

# Interested parties
@DiegoPino and any other @Islandora/7-x-1-x-committers

### CODE REVIEW

**Review by DiegoPino — APPROVED** [review]
This is a simple fix that should not affect running code in php 5.x. Also drupal does it all the time in core. You have my approval on this @ruebot. Will give this 24 hrs

**Comment by DiegoPino** [comment]
@ruebot i'm having trouble merging this. Tried 11 days ago when Github went crazy(unicorns) but today can't pass the "merge attempt failed".

**Comment by adam-vessey** [comment]
@DiegoPino @ruebot Just tried to merge here too, no go: Screenshot

**Comment by adam-vessey** [comment]
Tried out the CLI commands, appears to merge fine there? (didn't actually push the merged code) Shrug?

**Comment by dannylamb** [comment]
i'll see if there's anything funny with the repo settings

**Comment by dannylamb** [comment]
That's our standard setup there.  I'm not sure what's going on.  I'll look into it.
screenshot from 2016-11-25 11-53-41

**Comment by dannylamb** [comment]
Whelp....
screenshot from 2016-11-25 11-56-04

**Review by dannylamb — APPROVED** [review]
Trying to approve myself to see if it works.

**Comment by dannylamb** [comment]
Nope... @adam-vessey if you've got it set up to push, just push it.  We'll see if it blocks that.

**Comment by adam-vessey** [comment]
Here goes nothing...

**Comment by adam-vessey** [comment]
Appears to have done the trick?

**Comment by dannylamb** [comment]
Let's keep an eye out for that again.  Pretty weird.

**Comment by dannylamb** [comment]
Thx @adam-vessey

**Comment by DiegoPino** [comment]
@adam-vessey gracias!

**Comment by dannylamb** [comment]
pobody's nerfect

**Pull request merged**
